website/content/en/status/report-2024-10-2024-12/pinephone.adoc

```diff
@@ -0,0 +1,15 @@
+=== Pinephone Pro Support
+Links: +
+link:https://codeberg.org/Honeyguide/freebsd-pinephonepro[Repository on Codeberg] URL: link:https://codeberg.org/Honeyguide/freebsd-pinephonepro[]
+
+Contact: Toby Kurien <toby@tobykurien.com>
+
+The project to port FreeBSD over to the Pinephone Pro is progressing. The aim of this project is to step by step support components of the Pinephone Pro in FreeBSD so that the device one day might be usable as a highly mobile FreeBSD device.
```

Comment:
```suggestion
The project to port FreeBSD over to the Pinephone Pro is progressing.
The aim of this project is to step by step support components of the Pinephone Pro in FreeBSD so that the device one day might be usable as a highly mobile FreeBSD device.
```